index.js

```diff
@@ -44,29 +44,25 @@ scannerCliArgs = "";
  * https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs
  */
 function initialSetup() {
-  // TODO: validate inputs
-  const inputs = {
-    severityThreshold: core.getInput("severity-threshold"),
-    strictlyEnforcedRules: core.getInput("strictly-enforced-rules"),
+  const scannerFlags = {
```

Comment:
These changes look great - thank you for the simplification!

Were you able to test this running in an action's runtime from your fork?
The testing of this project has been quite challenging - it involves creating a tag and a release, then running the test-release action in a separate repo 😵‍💫

If you weren't able to test in an action's runtime, I can create a separate "release" branch for these pending changes to be bundled into a release and tested in a separate repo.

Reply:
I also think testing can be much simplified - it should be relatively simple to set up Jest tests for this repo to unit test all changes properly without even having to run the action locally. You certainly should not need to be running things in a separate repo.

Reply:
I was able to test this by creating a separate branch, tag, and release which included these changes and execute those in a separate test repository.

<img width="1084" alt="image" src="https://user-images.githubusercontent.com/18402464/212391965-09f9d162-7168-4567-b5d3-1d45c2ac993e.png">

<img width="552" alt="image" src="https://user-images.githubusercontent.com/18402464/212391914-7017bdfe-9a7d-47f1-af55-a37c4c83d19b.png">